pkg/R/SQLContext.R

```diff
@@ -36,12 +36,14 @@ jsonFile <- function(sqlCtx, path) {
 #' @export
 
 # TODO: Implement saveasParquetFile and write examples for both
-parquetFile <- function(sqlCtx, path) {
-  # Allow the user to have a more flexible definiton of the text file path
-  path <- normalizePath(path)
-  # Convert a string vector of paths to a string containing comma separated paths
-  path <- paste(path, collapse=",")
-  sdf <- callJMethod(sqlCtx, "parquetFile", path)
+parquetFile <- function(sqlCtx, ...) {
+  # Iterate through paths
+  paths <- list(...)
+  for (i in seq_along(paths)) {
+    # Allow the user to have a more flexible definiton of the text file path
+    paths[[i]] <- normalizePath(paths[[i]])
+  }
```

Comment:
This looks good. Interesting that our standard approach of passing a comma-separated string of paths doesn't work in this case. @shivaram is this a new change that might affect our other read methods? I think almost all of them use the `paste(path, collapse=",")` trick to handle multiple files.